conf/defaults.ini

```diff
@@ -106,6 +106,25 @@ path = grafana.db
 # For "sqlite3" only. cache mode setting used for connecting to the database
 cache_mode = private
 
+#################################### Login ###############################
+
+[login]
+
+# Login cookie name
+cookie_name = grafana_session
+
+# If you want login cookies to be https only. default is false
+cookie_secure = false
```

Comment:
This should probably be a global setting since all cookies set in backend should use this setting

Reply:
Good point. I fix.